### PR TITLE
fix typo and improve several files in .operations folder

### PR DESCRIPTION
**Pull request by Miigon**
fixed typos, and improved grammar/phrases in:
* CONTRIBUTING.md
* writing-guidelines.md
* operations-manual.md
* .common-answers.md

Better Chinese translation for:
* writing-guidelines.chinese.md

### CODE REVIEW

File: .operations/CONTRIBUTING.md [hunk 1]
```diff
@@ -18,10 +18,10 @@ Our content writing guidelines [can be found here](https://github.com/goldbergyo
 
 ### Handling issues and PRs
 
-- Greet the person who contribute out of a good will
-- If a strighforward fix is suggested (e.g., TYPO, linting fix, grammaer improvement, etc) - Approve and merge
-- When a the proposed PR deals with a more opionionated topic - Encourage at least 2 reviewer and let the PR land for a few days to let more folks chime-in
-- Add the contributor to the contributions credit list with this command: `@all-contributors please add @username for content`
+- Great the contributor with kindness
```
Comment (goldbergyoni) [inline .operations/CONTRIBUTING.md:21]: It's actually important to remind that the contributor is volunteering (WHY should we be kind)

File: .operations/operations-manual.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # Operations Manual - Organizing and Maximizing Our Work
-Building a community and knowledge by efficiently handling issues
+Building knowledge and a community by efficiently handling issues
 
-## Handling issues and PR
+## Handling issues and PRs
 
 <br/>
-In a nutshell, every issue and PR should get tagged by one of our core team and routed to the person who specializes in the related topic. This person then, will warmly welcome and kick the discussion shortly (hopefully within 48 hours). The goal of each issue/PR is to learn new thing that might improve our repo and try joining the opener to our forces.
+In a nutshell, every issue and PR should get tagged by one of our core team and routed to the person who specializes in the related topic. This person then, will warmly welcome the contributor and then kick off the discussion (hopefully within 48 hours). The goal of each issue/PR is to learn new thing that might improve our repo and try to include the contributor in our army.
```
Comment (goldbergyoni) [inline .operations/operations-manual.md:7]: Isn't army a militant word?

**Review by goldbergyoni — APPROVED** [review]
Great improvements!

**Comment by goldbergyoni** [comment]
Thank you @Miigon 💪✨✨🌻

@all-contributors please add @Miigon for content

File: .operations/operations-manual.md (hunk 2 above)
Comment (Miigon) [reply .operations/operations-manual.md:7]: Yes but so is force in my opinion. Maybe a better word could be used instead.
I'm thinking "family" but it lacks the original feeling of "being unstoppable" 😂
Now thinking of it again I actually prefer "force" if no better alternative could be found.

File: .operations/CONTRIBUTING.md (hunk 1 above)
Comment (Miigon) [reply .operations/CONTRIBUTING.md:21]: Oops, accidentally made a typo for 'Greet' here. 😅

"Greet the contributor with kindness for their hard work and generous contribution?"

**Pull request merged**